areas/sql/src/AzureMcp.Sql/Commands/Database/DatabaseShowCommand.cs

```diff
@@ -53,13 +53,6 @@ public override async Task<CommandResponse> ExecuteAsync(CommandContext context,
                 options.Subscription!,
                 options.RetryPolicy);
 
-            if (database == null)
```

Comment:
Why removed?

Reply:
The `GetDatabaseAsync` does not return null when the database is missing; instead, it throws a `KeyNotFoundException`. As a result, the `GetErrorMessage` and `GetStatusCode` methods are updated to handle this scenario as a 404 error.